Deploy/stacks/UK-building-retrofit/StackDeployment/README.md

```diff
@@ -117,21 +116,32 @@ This query retrieves the average energy efficiency ratings for all buildings in
 
 Further query templates can be found in the Deploy/stacks/UK-building-retrofit/StackDeployment/SPARQL queries directory, covering a range of metrics such as energy consumption, fuel poverty rates, and public health data.
 
+## 5. Feature Info Agent
+The [Feature Info Agent] serves as an access point for the visualisation, enabling queries about data for individual features (e.g., specific geographical locations). In this case, the Feature Info Agent retrieves energy performance and structural descriptions for buildings by querying the Ontop endpoint using the building's IRI.  
+
+The queries and config files can be found in [fia_quires]. Navigate to [services], open [feature-info-agent.json] and under the `Mounts` section, modify `Source` based on where the queries and config files are located at. 
+
 <!-- Links -->
 [allows you to publish and install packages]: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry#authenticating-to-github-packages
 [Container registry on Github]: https://github.com/orgs/cambridge-cares/packages
-[Create SSH key]: https://docs.digitalocean.com/products/droplets/how-to/add-ssh-keys/create-with-openssh/
-[Environment Agency]: https://environment.data.gov.uk/flood-monitoring/doc/reference
+[SSH key]: https://docs.digitalocean.com/products/droplets/how-to/add-ssh-keys/create-with-openssh/
 [forwarding the port]: https://code.visualstudio.com/docs/remote/ssh#_forwarding-a-port-creating-ssh-tunnel
 [OS Features API]: https://api.os.uk/features/
 [personal access token]: https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token
 [VSCode via SSH]: https://code.visualstudio.com/docs/remote/ssh
 [Upload SSH key]: https://docs.digitalocean.com/products/droplets/how-to/add-ssh-keys/to-existing-droplet/
 [MetOffice My Account]: https://register.metoffice.gov.uk/MyAccountClient/account/view
+[The World Avatar]: https://github.com/cambridge-cares/TheWorldAvatar.git
+[git clone]: https://git-scm.com/docs/git-clone
+[SSH tunnel]: https://code.visualstudio.com/docs/remote/tunnels
+[Feature Info Agent]: https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Agents/FeatureInfoAgent
 
 <!-- Stack references -->
 [common stack scripts]: https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Deploy/stacks/dynamic/common-scripts
 [Stack data uploader]: https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Deploy/stacks/dynamic/stack-data-uploader
 [Stack manager]: https://github.com/cambridge-cares/TheWorldAvatar/blob/main/Deploy/stacks/dynamic/stack-manager/README.md
+[fia_quires]: ./Stack-manager/inputs/data/fia-queries
```

Comment:
Please fix the typo.

Reply:
Sorry for the mistake, this was fixed in commit [4eb60b4acc6e48d628673b8595c1cd90a6c490f9]